dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -69,4 +71,24 @@ public ClassPathResult(
   ImmutableList<UnresolvableArtifactProblem> getArtifactProblems() {
     return artifactProblems;
   }
+
+  /** Returns text describing dependency paths to {@code jars} in the dependency tree. */
+  public String formatDependencyPaths(Iterable<Path> jars) {
+    StringBuilder message = new StringBuilder();
+    for (Path jar : jars) {
+      ImmutableList<DependencyPath> dependencyPaths = getDependencyPaths(jar);
+      checkArgument(dependencyPaths.size() >= 1, "%s is not in the class path", jar);
+
+      message.append(jar.getFileName() + " is at:\n");
+
+      int otherCount = dependencyPaths.size() - 1;
+      message.append("  " + dependencyPaths.get(0) + "\n");
+      if (otherCount == 1) {
+        message.append("  and 1 dependency path.\n");
+      } else if (otherCount > 1) {
+        message.append("  and other " + otherCount + " dependency paths.\n");
```

Comment:
and 163 other dependency paths
instead
and other 163 dependency paths

Reply:
Updated.